### PR TITLE
Format changes to README

### PR DESCRIPTION
**Pull request by c10ce**
Improving the display of the README file with list format and header formatting for Georgia Institute of Technology Course Critique. 

Also splitting out the easter egg code into its own file (so we can include it if wanted or leave it out for bandwidth)

### CODE REVIEW

File: js/easter.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    //Random clippy
+    var agents = ["Bonzi", "Genius", "Clippy", "F1", "Genie", "Links", "Merlin", "Rover"];
+    agents = agents[Math.floor(Math.random()*agents.length)];
+    clippy.load(agents, function(agent) {
+      agent.show();
+      agent.speak("You Found Me! How about we sing a song?   ");
+      setTimeout(function() {
+        if(window.HTMLAudioElement) {
+          var snd = new Audio('');
+          if(snd.canPlayType('audio/mp3')) {
```
Comment (saiarcot895) [inline js/easter.js]: Should the if-statement order be switched so it picks the file with the smaller size whenever possible?

**Comment by c10ce** [comment]
@saiarcot895 I did switch the if-statements around (to put error/problems first). Not sure which file you mean, though; are you referring to the OGG/MP3, or the agents? In either case, I'm not sure it'll mean much in the grand scheme of things to optimize those (especially this code would likely not be called frequently)

**Comment by saiarcot895** [comment]
@chrisforrence I meant the OGG/MP3 blocks.  (I should have added the line number range.)

**Comment by c10ce** [comment]
@saiarcot895 Ah, I see what you mean. I'm not as worried about it; the file size difference between the two is less than 100 bytes, it's a low chance the code gets executed, and the MP3 format has a higher likelihood of being supported in available browsers (IE+Chrome+Safari+Firefox vs Chrome+Firefox+Opera). If you'd like to switch them, feel free to make a pull request against my branch, and I'll merge it in

**Pull request merged**